lib/HttpKernel/CacheWarmer/PimcoreCoreCacheWarmer.php

```diff
@@ -46,7 +46,12 @@ public function warmUp($cacheDir): array
 
         if (\Pimcore::isInstalled()) {
             try {
+                $cache = \Pimcore\Cache::isEnabled();
+                \Pimcore\Cache::disable();
                 $this->dataObjectClasses($classes);
+                if ($cache) {
+                    \Pimcore\Cache::enable();
+                }
```

Comment:
Will the cache be re-enabled if `$this->dataObjectClasses($classes);` throws? If not, maybe add a `finally` and move the `::enable()` there so it may be re-enabled regardless?

Reply:
Nice suggestion, thanks @NiklasBr!